src/lib/components/table/cells/pile-cell.svelte

```diff
@@ -0,0 +1,56 @@
+<script lang="ts" context="module">
+  export interface PileComponentProps {
+    components: PileComponent[];
+    maxItems: number;
+  }
+
+  export const dripListIcon = (dripList: DripListBadgeFragment) => ({
+    component: DripListBadge,
+    props: {
+      dripList,
+      showOwner: false,
+      showName: false,
+      outline: true,
+    },
+  });
+
+  export const projectIcon = (project: ProjectAvatarFragment) => ({
+    component: ProjectAvatar,
+    props: {
+      project,
+      outline: true,
+      size: 'small',
+    },
+  });
+
+  export const addressIcon = (address: string) => ({
+    component: IdentityBadge,
+    props: {
+      address,
+      showIdentity: false,
+      outline: true,
+      size: 'medium',
+      disableTooltip: true,
+    },
+  });
```

Comment:
why are these exported from `pile-cell`? the pile cell can accept any configuration for `Pile`, so defining these seems like it should be part of the parent component.